src/script/createChallenge.js

```diff
@@ -16,16 +16,14 @@ function createChallenge(challenge) {
     createStars(challengeItem, challenge.rating);
 
     participants.textContent = challenge.minParticipants + "-" + challenge.maxParticipants + " participants";
-    participants.dataset.minParticipants = challenge.minParticipants;
-    participants.dataset.maxParticipants = challenge.maxParticipants;
     participants.classList.add("challenge-meta");
     challengeItem.appendChild(participants);
 
     challengeDescription.textContent = challenge.description;
     challengeDescription.classList.add("challenge-description");
     challengeItem.appendChild(challengeDescription);
 
-    createButton(challengeItem, challenge.type, challenge.title, challenge.id);
+    createButton(challengeItem, challenge.type, challenge.title, challenge.id, challenge.minParticipants, challenge.maxParticipants);
```

Comment:
At some point, it might make sense to change this function so that you can just pass the entire `challenge` object to it, instead of passing five attributes from the object as separate arguments.